lib/plugins/graphite/data-generator.js

```diff
@@ -50,6 +50,14 @@ function keyPathFromMessage(message, options, includeQueryParams, alias) {
     // add the group of the summary message
     typeParts.splice(1, 0, graphiteUtil.toSafeKey(message.group));
   }
+  // if we are sending run metrics and we have an iteration
+  if (
+    options.perIteration &&
```

Comment:
Options here are generic options and not Graphite specific right?

Reply:
Looking at the code again, I thought I had tested this out. It looks like the data-generator gets the options pre-merge and its the whole object and not graphite specific.